### DISCUSSION TITLE
Dynamic prompts when training embeddings

### DISCUSSION THREAD

**Discussion by EmileVDBerghe** [adieyal/sd-dynamic-prompts#102] [Q&A] [answered]
Is it possible to use this script for previews while training embeddings? There is a checkbox "_Read parameters (prompt, etc...) from txt2img tab when making previews_" but it just uses the raw prompt without the script. I would love to use the dynamic prompts to give me applied previews to check in on the progress of how my trained embedding is progressing.

**Comment by adieyal** [comment, marked as answer]
I'll take a look - you can follow the issue here: https://github.com/adieyal/sd-dynamic-prompts/issues/108

**Reply by EmileVDBerghe** [reply]
Thank you!
